code-coverage/pom.xml

```diff
@@ -257,11 +257,17 @@
                       <fileset dir="../ArjunaJTA/jms/target/${surefire.reports.directory}" includes="**/*.xml"/>
                       <fileset dir="../tools/target/${surefire.reports.directory}" includes="**/*.xml"/>
 
+                      <!--
                       <fileset dir="../ArjunaJTS/jts/target/idlj-surefire-reports" includes="**/*.xml"/>
+                      -->
                       <fileset dir="../ArjunaJTS/jts/target/idlj-openjdk-surefire-reports" includes="**/*.xml"/>
+                      <!--
                       <fileset dir="../ArjunaJTS/orbportability/target/idlj-surefire-reports" includes="**/*.xml"/>
+                      -->
                       <fileset dir="../ArjunaJTS/orbportability/target/idlj-openjdk-surefire-reports" includes="**/*.xml"/>
+                      <!--
                       <fileset dir="../ArjunaJTS/jtax/target/${surefire.reports.directory}" includes="**/*.xml"/>
+                      -->
```

Comment:
Yeah, I think those should have been removed in JBTM-3590...So WDYT if we remove them in this PR? (e.g. renaming this commit on the line `[JBTM-3590] left-over`)

Reply:
They have been removed/disabled (I used `<!-- .. -->` XML style comments).

I left them as comments so that its clear we don't produce reports for the modules, I want them there as place holders just in case we are not including some tests in the report.

Reply:
IIRC, the `idlj` module was removed completely, hence `idlj-surefire-reports` cannot be produced any longer. These lines might introduce confusion as they point to a module that is only available in the git history.

Reply:
The problem is that the JACOCO profile was never ran with the PRs that removed the code coverage profiles so we have only discovered the problem now. Guessing is the wrong option so I prefer to comment them out and check later whether we have the correct set of code coverage reports.

Reply:
OK, that makes sense. I will create a JIRA/add this to my ToDo list

Reply:
[JBTM-3651](https://issues.redhat.com/browse/JBTM-3651)